githooks/post-checkout

```diff
@@ -24,3 +24,5 @@ if [ "$3" == "1" ]; then
     fi
 fi
 
+mkdir -p bin
+ln -sf scripts/drtprod bin/drtprod
```

Comment:
I believe symlinks are resolved relative to the directory containing the link, so this would be `ln -sf ../scripts/drtprod bin/drtprod` in that case?

Reply:
You're right, sorry about that.